mungegithub/Dockerfile-publisher

```diff
@@ -24,6 +24,8 @@ RUN apt-get update \
 
 ENV GOPATH="/go-workspace"
 ENV PATH="${GOPATH}/bin:/usr/local/go/bin:${PATH}"
+ENV GIT_COMMITTER_EMAIL="k8s-publish-robot@users.noreply.github.com"export GIT_COMMITTER_NAME="Kubernetes Publisher"
```

Comment:
Is this line right? It looks like mingled.

Reply:
surprising that it does not fail. rebase or copy&paste error

Reply:
fixed